install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -30,6 +33,7 @@ data:
     # from istio Pilot. Lower refresh delay results in higher CPU
     # utilization and potential performance loss in exchange for faster
     # convergence. Tweak this value according to your setup.
+    # Deprecated: using push.
     rdsRefreshDelay: {{ .Values.global.refreshInterval }}
```

Comment:
please remove.. We are not supporting any form of v1 in 1.0. So, having this here merely adds to confusion.

Reply:
Done.